homeworks/roman.pavliuk_RomanPvlk/7-arrays-loops/arrays.js

```diff
@@ -0,0 +1,108 @@
+const arrayOne = [3, 0, -5, 1, 44, -12, 3, 0, 0, 1, 2, -3, -3, 2, 1, 4, -2, -3, -1];
+const arrayTwo = [-1, -8, -2];
+const arrayThree = [1, 7, 3];
+const arrayFour = [1, undefined, 3, 5, -3];
+const arrayFive = [1, NaN, 3, 5, -3];
+
+function maxNumberOfArr(array) {
+    let maxNumber = array[0];
+    for (let i = 0; i < array.length; i++) {
```

Comment:
Here, and in the next func you can start iteration from 2nd array item if you set 1st item as init value